src/jit/compiler.h

```diff
@@ -604,13 +638,19 @@ class  LclVarDsc
     bool                lvIsMultiregStruct()
```

Comment:
IMO, this routine should be named as lvIsMultiRegPassedStruct() to be explicit and clear.